### DISCUSSION TITLE
Jenkins seed-job error

### DISCUSSION THREAD

**Discussion by chennamjaya** [learndevopsonline/learndevopsonline#1659] [Q&A]
I am getting the following error while trying to run the seed-job in jenkins
![Screenshot 2023-12-05 132327](https://github.com/learndevopsonline/learndevopsonline/assets/62970385/cfb91040-d047-4b51-b018-52802d6bd063)

**Comment by sreedharm07** [comment]
you dint declared file_name variable in your code
you are calling file_name variable in jobs.xml file but dint declared in code
please declare file_name variable(file_name: Jenkinsfile) in vars

**Reply by chennamjaya** [reply]
I have declared the variable_still getting a different error:
![Uploading Screenshot 2023-12-05 143443.png…]()

**Reply by chennamjaya** [reply]
![Screenshot 2023-12-05 143443](https://github.com/learndevopsonline/learndevopsonline/assets/62970385/3f9cb5e2-8f1b-40ff-943e-09ed3539ae23)

**Reply by sreedharm07** [reply]
ping your jenkins credentials

**Reply by chennamjaya** [reply]
user: admin
password: admin1234

**Reply by sreedharm07** [reply]
error was with your job.xml file.
just remove job.xml file and create new one 

create one job manually and go to path and copy paste in job.xml file and then run again
it will work

**Comment by chennamjaya** [comment]
Deleted the existing jobs and created a new job manually. Copied the config.xml from the path and pasted in the job.xml it worked.
